resources/views/emails/quiz/results.blade.php

```diff
@@ -0,0 +1,9 @@
+<x-mail::message>
+    # {{ __('Quiz results') }}
+
+    {{ __('Congraturations') . ', ' . $name . '!' }}
+    {{ __('You have successfully completed course') . ' ' . $course . ' ' . __('and passed the quiz') . '.' }}
```

Comment:
These should all use the string interpolation. Also there is a typo Congraturations => Congratulations.

```suggestion
    {{ __('Congratulations, :name!', ['name' => $name]) }}
    {{ __('You have successfully completed course :course and passed the quiz.', ['course' => $course] }}
```